kie-server-parent/kie-server-controller/kie-server-controller-standalone/src/main/resources/META-INF/services/org.kie.server.controller.api.storage.KieServerTemplateStorage

```diff
@@ -0,0 +1 @@
+org.kie.server.controller.impl.storage.FileBasedKieServerTemplateStorage
```

Comment:
It would be good to place this file into same module where FileBasedKieServerTemplateStorage is located. It will be less error prone in case of future changes.